features/events/daemon/pom.xml

```diff
@@ -88,6 +88,11 @@
       <scope>test</scope>
       <type>pom</type>
     </dependency>
+    <dependency>
+      <groupId>org.opennms.features.collection</groupId>
+      <artifactId>org.opennms.features.collection.persistence.osgi</artifactId>
```

Comment:
We should add this dependency in `opennms-rrdtool-api`, `opennms-rrd-jrobin` and `org.opennms.features.newts` instead of adding to every module that depends on one of these.